src/events/guild/member/guildMemberAdd.ts

```diff
@@ -33,10 +33,10 @@ export default class extends Event {
 
 		const mute: Record<string, any> = await getLatestMute(member, member.guild.id);
 		if (mute) {
-			const muteRole: Snowflake = <Snowflake>await databaseManager.getMuteRole(<Snowflake>member.guild?.id);
+			const muteRole: Snowflake | null = await databaseManager.getMuteRole(<Snowflake>member.guild.id);
```

Comment:
```suggestion
			const muteRole: Snowflake | null = await databaseManager.getMuteRole(member.guild.id);
```
If `guild` is null-safe then we don't need to case this either